packages/scandipwa/src/component/RecentlyViewedWidget/RecentlyViewedWidget.container.js

```diff
@@ -25,6 +27,11 @@ export const mapDispatchToProps = (dispatch) => ({});
 
 /** @namespace Component/RecentlyViewedWidget/Container */
 export class RecentlyViewedWidgetContainer extends PureComponent {
+    static propTypes = {
+        recentProducts: PropTypes.object.isRequired,
```

Comment:
`PropTypes.object` is forbidden in our eslint configuration. You should use `PropTypes.shape()` instead.

Reply:
Changed to ```PropTypes.objectOf(ItemsType).isRequired```, because key is not static.

Reply:
Okay!